.travis.yml

```diff
@@ -6,6 +6,8 @@ node_js:
   # https://github.com/nodejs/LTS
   - "6" # ends April 2019
```

Comment:
Could you remove these two while you are at it? They are either at end of life or nearing it. No need to run the tests longer than needed.